src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1938,6 +1938,71 @@ private void queryLoans(int limit)
     });
   }
 
+  @Test
+  public void canGetAnonymizedLoan() throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    InventoryItemResource item = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource jessica = usersFixture.jessica();
+
+    IndividualResource individualResource = loansFixture.checkOutByBarcode(item,
+      jessica, new DateTime(2018, 4, 21, 11, 21, 43, DateTimeZone.UTC));
+
+    JsonObject savedLoan = loansStorageClient.get(individualResource.getId())
+      .getResponse().getJson();
+
+     savedLoan.remove("userId");
+
+     loansStorageClient.replace(individualResource.getId(), savedLoan);
+
+     JsonObject loan = loansClient.get(individualResource.getId()).getJson();
+
+     loanHasPatronGroupProperties(loan, "Regular Group");
+  }
+
+  @Test
+  public void canGetMultipleAnonymizedLoans() throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    InventoryItemResource firstItem = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource jessica = usersFixture.jessica();
+    IndividualResource firstLoan = loansFixture.checkOutByBarcode(firstItem,
+      jessica, new DateTime(2018, 4, 21, 11, 21, 43, DateTimeZone.UTC));
+
+
```

Comment:
Why are there two blank lines here? I think one is sufficient to separate the sections

Reply:
Thank you